vendor/engines/secure_rooms/app/presenters/secure_rooms/scan_response_presenter.rb

```diff
@@ -0,0 +1,40 @@
+module SecureRooms
+
+  class ScanResponsePresenter
+
+    def initialize(user, access_verdict, accounts)
+      @user = user
+      @access_verdict = access_verdict
+      @accounts = accounts
+    end
+
+    def response
+      {
+        status: status_for_code(@access_verdict.result_code),
+        json: {
+          # TODO: (#140895375) return actual tablet_identifier
+          tablet_identifier: "abc123",
+          name: @user.full_name,
+          response: @access_verdict.result_code,
+          reason: @access_verdict.reason,
+          accounts: SecureRooms::AccountPresenter.wrap(@accounts),
```

Comment:
unless it's specifically unwanted I'd just leave it in to reduce complexity for now